include/robot_localization/ros_filter_utilities.h

```diff
@@ -97,6 +98,72 @@ void stateToTF(const Eigen::VectorXd &state, tf2::Transform &stateTF);
 //!
 void TFtoState(const tf2::Transform &stateTF, Eigen::VectorXd &state);
 
+
```

Comment:
It's no big deal one way or the other, but this could also be promoted to its own header.